.travis.yml

```diff
@@ -14,23 +14,13 @@ services:
 
 matrix:
   include:
-  - env: ARGS="--resolver lts-6  --stack-yaml stack_lts-10.yaml" BACKEND=none
-  - env: ARGS="--resolver lts-7  --stack-yaml stack_lts-10.yaml" BACKEND=none
-  - env: ARGS="--resolver lts-9  --stack-yaml stack_lts-10.yaml" BACKEND=none
-  - env: ARGS="--resolver lts-10 --stack-yaml stack_lts-10.yaml" BACKEND=none
-  - env: ARGS="--resolver lts-11" BACKEND=none
-  - env: ARGS="--resolver lts-12" BACKEND=none
-  - env: ARGS="--resolver lts-13" BACKEND=none
-
-  - env: ARGS="--resolver lts-12" BACKEND=sqlite
-  - env: ARGS="--resolver lts-12" BACKEND=mongodb
-  - env: ARGS="--resolver lts-12" BACKEND=postgresql
-  - env: ARGS="--resolver lts-12" BACKEND=mysql
-
-  - env: ARGS="--resolver lts-13" BACKEND=sqlite
-  - env: ARGS="--resolver lts-13" BACKEND=mongodb
-  - env: ARGS="--resolver lts-13" BACKEND=postgresql
-  - env: ARGS="--resolver lts-13" BACKEND=mysql
+  - env: ARGS="--resolver lts-6  --stack-yaml stack_lts-10.yaml"
+  - env: ARGS="--resolver lts-7  --stack-yaml stack_lts-10.yaml"
+  - env: ARGS="--resolver lts-9  --stack-yaml stack_lts-10.yaml"
+  - env: ARGS="--resolver lts-10 --stack-yaml stack_lts-10.yaml" 
+  - env: ARGS="--resolver lts-11"
+  - env: ARGS="--resolver lts-12"
+  - env: ARGS="--resolver lts-13"
```

Comment:
Hooray for simpler CI!

Reply:
They're all resolvers for distinct GHC versions. 7.10.3, 8.0.2, 8.2, 8.4, and 8.6 are represented here. More resolvers seemed redundant to me.